tenzir/functional-test/lib/bats-tenzir/src/fixtures.bash

```diff
@@ -1,10 +1,26 @@
 # SPDX-FileCopyrightText: (c) 2023 The Tenzir Contributors
 # SPDX-License-Identifier: BSD-3-Clause
 
+export_default_paths() {
+  BATS_TEST_DATADIR="$(realpath $(dirname ${BATS_TEST_DIRNAME}))"
+  # Look for the first folder called `functional-test/` that is a parent
+  # of the location of the test file.
+  export BATS_TENZIR_DATADIR="${BATS_TEST_DATADIR%%/functional-test/*}/data"
+
+  export BATS_TENZIR_INPUTSDIR="${BATS_TENZIR_DATADIR}/inputs"
+  export BATS_TENZIR_QUERIESDIR="${BATS_TENZIR_DATADIR}/queries"
+  export BATS_TENZIR_MISCDIR="${BATS_TENZIR_DATADIR}/misc"
+  export BATS_TENZIR_CONFIGDIR="${BATS_TENZIR_DATADIR}/config"
+}
 
 export_default_node_config() {
-  export TENZIR_DB_DIRECTORY="${BATS_TEST_TMPDIR}/db"
+  # Normalize the environment unless `BATS_TENZIR_KEEP_ENVIRONMENT` is set.
+  if [[ ! -n ${BATS_TENZIR_KEEP_ENVIRONMENT} ]]; then
+    unset $(printenv | grep -o '^TENZIR[^=]*' | paste -s -)
+  fi
+
   export TENZIR_BARE_MODE=true
+  export TENZIR_DB_DIRECTORY="${BATS_TEST_TMPDIR}/db"
```

Comment:
I think this needs to be a new directory every time the function runs. Otherwise, we risk problems with tests running in parallel that use import or export.

Reply:
`BATS_TEST_TMPDIR` is a bats built-in variable that is unique per test run (according to the docs at least, i didn't actually try running in parallel yet)

We could move it to `setup_node()` so that its impossible to accidentally forget to reset it, but on the other hand then it would also be impossible to overwrite it for custom tests.

Reply:
Ah, I assumed that it was unique per suite. Makes sense, then we don't need to do anything here.